### ISSUE TITLE
Organization of Controls vs Typedefs

### ISSUE THREAD

**Issue by Monkeymerlot** [levylabpitt/AFM-Lithography#39]
@ciozi137 I had intended that the controls folder be used for custom made controls (ie the flat UI 2.0 ones) instead of type defs. I do agree though that the type defs folder should be brought out to the lvsrc folder.
![image](https://user-images.githubusercontent.com/37567672/86422290-baa41880-bcaa-11ea-97d5-bb25434210be.png).
What do you think?

**Comment by ciozi137** [comment]
I agree it can go into a typedef folder. It just shouldn’t be in subVIs

**Comment by Monkeymerlot** [comment]
I agree. I think what I was going at is how do you think we should organize stuff? I am not a big fan of stuff being in the subVIs folder in general. Everything is technically a subVI.

**Comment by ciozi137** [comment]
Except for the top level VI...

I do like sub folders that are a better descriptor of their contents than “subVi”. Can be public, private, utilities, API, framework, override, subsystems,  etc

**Comment by Monkeymerlot** [comment]
So basically we should move the folders out from the subVI folder?

**Comment by ciozi137** [comment]
I think that would be fine.

**Comment by Monkeymerlot** [comment]
Ok Ill do that later tonight after everyone is done working on it.

**Comment by Monkeymerlot** [comment]
I tried to do this the other day and LabVIEW started throwing a fit. I will do this again another time more slowly.

**Comment by ciozi137** [comment]
Ok. Do you think it makes sense to clean up the branches at the same time?

**Comment by Monkeymerlot** [comment]
I think we should do it when all branches are closed except develop, and even then only when it is even with Master. I dont even want to think about the issues merging that would be.

**Comment by ciozi137** [comment]
Do you mean fix the branches first and then fix the folder structure?

I would be ok merging everything / deleting branches, till we are down to one branch (master), then fix the folders, and finally create the branches again

**Comment by Monkeymerlot** [comment]
I have already started developing on the develop branch as it is. We will need to probably do a build again sometime this week to fix the Drag and Drop functionality that got removed when I replaced the listbox control. Right before the build seems like a good place to ensure that everything is merged etc. and then we can build. That way if we need to do further bug fixes the repository wont be totally chaotic.

**Comment by Monkeymerlot** [comment]
This is slowly being implemented as we finish #16
